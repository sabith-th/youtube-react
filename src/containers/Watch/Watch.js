import React from "react";
import { RelatedVideos } from "../../components/RelatedVideos/RelatedVideos";
import { Video } from "../../components/Video/Video";
import { VideoInfoBox } from "../../components/VideoInfoBox/VideoInfoBox";
import { VideoMetadata } from "../../components/VideoMetadata/VideoMetadata";
import "./Watch.scss";

export class Watch extends React.Component {
  render() {
    return (
      <div className="watch-grid">
        <Video className="video" id="6ZfuNTqbHE8" />
        <VideoMetadata className="metadata" viewCount={10000000} />
        <VideoInfoBox className="video-info-box" />
        <div
          className="comments"
          style={{ width: "100%", height: "100px", background: "#9013FE" }}
        >
          Comments
        </div>
        <RelatedVideos className="related-videos" />
      </div>
    );
  }
}
