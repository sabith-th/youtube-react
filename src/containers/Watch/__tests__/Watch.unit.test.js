import { shallow } from "enzyme";
import React from "react";
import { Watch } from "../Watch";

describe("Watch", () => {
  test("renders", () => {
    const wrapper = shallow(<Watch />);
    expect(wrapper).toMatchSnapshot();
  });
});
