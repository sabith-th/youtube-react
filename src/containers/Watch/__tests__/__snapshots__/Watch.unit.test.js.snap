// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Watch renders 1`] = `
<div
  className="watch-grid"
>
  <Video
    className="video"
    id="6ZfuNTqbHE8"
  />
  <VideoMetadata
    className="metadata"
    viewCount={10000000}
  />
  <VideoInfoBox
    className="video-info-box"
  />
  <div
    className="comments"
    style={
      Object {
        "background": "#9013FE",
        "height": "100px",
        "width": "100%",
      }
    }
  >
    Comments
  </div>
  <RelatedVideos
    className="related-videos"
  />
</div>
`;
